src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml

```diff
@@ -162,7 +162,7 @@
                         </TextBlock>
                     </StackPanel>
              </StackPanel>
-             <StackPanel x:Name="LowConfidenceSpace" Orientation="Horizontal" VerticalAlignment="Center" HorizontalAlignment="Left" Margin="0,10,0,10"
+             <StackPanel x:Name="LowConfidenceSpace" MouseLeftButtonUp="ShowLowConfidenceResults" Orientation="Horizontal" VerticalAlignment="Center" HorizontalAlignment="Left" Margin="0,10,0,10"
```

Comment:
Do we also have an option to revert this? That is, hide `Low Confidence` results after the user have expanded the list?

Reply:
Not in that window. When the node autocomplete window is opened again, it will be back to starting behaviour.